### DISCUSSION TITLE
iOS suppot

### DISCUSSION THREAD

**Discussion by karnilamit** [vpnhood/VpnHood#28] [Q&A] [answered]
Hi,

Do you plan to add support for iOS devices?

Thanks,
Amit

**Comment by trudyhood** [comment, marked as answer]
We have plan for it, but after VpnHood get more mature.
VpnHood is built on .NET Standard, and we just need to add an iOS provider to it and deal with Apple Store.

**Reply by karnilamit** [reply]
Great. Thanks.

**Reply by Tenera** [reply]
Hi, Is there some kind of roadmap, so we can anticipate when we could expect the iOS implementation?

**Reply by trudyhood** [reply]
Most VPN hood code is written in .NET standard which is portable, and just a class and app container needed to be written for iOS. It is not a big deal but unfortunately, we have to learn how to do it. I wish someone join us and do that. Currently, we have no milestone for that.

**Reply by PeterVD-Lansweeper** [reply]
I understand. :-) 
If I could, I would help, but I don't know anything about it either.
